concourse/client/api.py

```diff
@@ -18,6 +18,7 @@
 import typing
 
 from ensure import ensure_annotations
+import requests
```

Comment:
nit: should come before `from ...`, looking at the other imports

Reply:
done using `importanize`